<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>BMPVolumeViewer</class>
  <widget class="QMainWindow" name="BMPVolumeViewer">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>1200</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>VTK BMP序列三维体渲染与切片查看</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QGroupBox" name="groupBox">
            <property name="title">
              <string>文件设置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
                <widget class="QLabel" name="label">
                  <property name="text">
                    <string>文件前缀:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QLineEdit" name="filePrefixEdit"/>
              </item>
              <item row="0" column="2">
                <widget class="QPushButton" name="browseButton">
                  <property name="text">
                    <string>浏览...</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                  <property name="text">
                    <string>文件名格式:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1" colspan="2">
                <widget class="QLineEdit" name="filePatternEdit">
                  <property name="text">
                    <string>%s%d.00um.bmp</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0" colspan="3">
                <widget class="QPushButton" name="loadButton">
                  <property name="text">
                    <string>加载图像序列</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
            <item>
              <widget class="QGroupBox" name="volumeGroupBox">
                <property name="title">
                  <string>三维体渲染</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                    <widget class="QVTKOpenGLNativeWidget" name="volumeWidget">
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="minimumSize">
                        <size>
                          <width>600</width>
                          <height>600</height>
                        </size>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
                <item>
                  <widget class="QGroupBox" name="xSliceGroupBox">
                    <property name="title">
                      <string>X方向切片 (平行于Z轴)</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_4">
                      <item>
                        <widget class="QVTKOpenGLNativeWidget" name="xSliceWidget">
                          <property name="minimumSize">
                            <size>
                              <width>300</width>
                              <height>300</height>
                            </size>
                          </property>
                          <property name="sizePolicy">
                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                              <horstretch>0</horstretch>
                              <verstretch>0</verstretch>
                            </sizepolicy>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <item>
                            <widget class="QLabel" name="xSliceLabel">
                              <property name="text">
                                <string>X切片位置: 0</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QSlider" name="xSliceSlider">
                              <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="minimum">
                                <number>0</number>
                              </property>
                              <property name="maximum">
                                <number>100</number>
                              </property>
                              <property name="value">
                                <number>50</number>
                              </property>
                            </widget>
                          </item>
                        </layout>
                      </item>
                    </layout>
                  </widget>
                </item>
                <item>
                  <widget class="QGroupBox" name="ySliceGroupBox">
                    <property name="title">
                      <string>Y方向切片 (平行于Z轴)</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                      <item>
                        <widget class="QVTKOpenGLNativeWidget" name="ySliceWidget">
                          <property name="minimumSize">
                            <size>
                              <width>300</width>
                              <height>300</height>
                            </size>
                          </property>
                          <property name="sizePolicy">
                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                              <horstretch>0</horstretch>
                              <verstretch>0</verstretch>
                            </sizepolicy>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                          <item>
                            <widget class="QLabel" name="ySliceLabel">
                              <property name="text">
                                <string>Y切片位置: 0</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QSlider" name="ySliceSlider">
                              <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="minimum">
                                <number>0</number>
                              </property>
                              <property name="maximum">
                                <number>100</number>
                              </property>
                              <property name="value">
                                <number>50</number>
                              </property>
                            </widget>
                          </item>
                        </layout>
                      </item>
                    </layout>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QGroupBox" name="controlGroupBox">
            <property name="title">
              <string>渲染控制</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
                <widget class="QLabel" name="opacityLabel">
                  <property name="text">
                    <string>不透明度:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSlider" name="opacitySlider">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="minimum">
                    <number>0</number>
                  </property>
                  <property name="maximum">
                    <number>100</number>
                  </property>
                  <property name="value">
                    <number>50</number>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="colorLabel">
                  <property name="text">
                    <string>颜色映射:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSlider" name="colorSlider">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="minimum">
                    <number>0</number>
                  </property>
                  <property name="maximum">
                    <number>100</number>
                  </property>
                  <property name="value">
                    <number>50</number>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="resetViewButton">
                  <property name="text">
                    <string>重置视图</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>